jicofo-common/src/main/kotlin/org/jitsi/jicofo/codec/CodecUtil.kt

```diff
@@ -30,6 +30,22 @@ class CodecUtil {
         fun createVideoPayloadTypeExtensions(
             options: OfferOptions = OfferOptions()
         ): Collection<PayloadTypePacketExtension> = buildList {
+            if (config.av1.enabled() && options.av1) {
+                // a:rtpmap:XXX AV1/90000
+                val av1 = createPayloadTypeExtension(config.av1.pt(), "AV1", 90000)
+                av1.addVideoExtensions(options, config.av1)
+                if (config.av1DependencyDescriptor.enabled()) {
+                    // https://aomediacodec.github.io/av1-rtp-spec/#dependency-descriptor-rtp-header-extension
+                    val dependencyDescriptorExt = RTPHdrExtPacketExtension()
+                    dependencyDescriptorExt.id = config.av1DependencyDescriptor.id().toString()
+                    dependencyDescriptorExt.uri = URI.create(
+                        "https://aomediacodec.github.io/av1-rtp-spec/#dependency-descriptor-rtp-header-extension"
+                    )
+                    av1.addChildExtension(dependencyDescriptorExt)
```

Comment:
I don't think this is right - RTPHdrExtPacketExtension is meant to be a child of Media, not Codec.

(I know that means that technically it would seem to apply to all codecs, but that's a limitation of Jingle/SDP.)

Reply:
thanks! this was not tested yet as I was porting this from the old codec config, still need to rebase the other changes for different repos